sympy/physics/vector/frame.py

```diff
@@ -621,13 +621,16 @@ def _rot(axis, angle):
                 from sympy.physics.vector.functions import kinematic_equations
                 q1, q2, q3 = amounts
                 u1, u2, u3 = symbols('u1, u2, u3', cls=Dummy)
```

Comment:
`u1, u2, u3` are being used in `else` condition hence they should be defined before `if-else` block.